frontend/src/app/features/in-app-notifications/entry/in-app-notification-entry.component.sass

```diff
@@ -28,7 +28,7 @@ $subject-font-size: 14px
     border-bottom: 1px solid var(--borderColor-default)
 
   &_selected
-    background: $ian-bg-selected-color
+    background: $ian-bg-selected-color !important
```

Comment:
Because of the `!important`, the hover effect styles below are completely ignored which is not what we want. I guess, you have to change ` $ian-bg-selected-hover-color` instead